CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 ### Fixed
+- Fixed being unable to build if the port used by `git://` is not open
 
 ### Removed
+- Removed `-c http.sslVerify=false` option from the command to clone postgres pldebugger repository in flowdb Dockerfile
```

Comment:
Drop this line?

Reply:
The fact that `-c http.sslVerify=false` was there in the first place implies that somebody needed it to get the clone to work. It works fine without this, in my experience, but I wonder whether there may be any cases where it's still necessary. I put this line in so that we have an explicit record in the changelog in case anybody sees a problem as a result of this in the future. I could be convinced otherwise, though, if you feel differently.